src/test/java/com/squareup/kotlinpoet/TypeSpecTest.kt

```diff
@@ -1122,6 +1122,57 @@ class TypeSpecTest {
         |""".trimMargin())
   }
 
+  @Test fun kdocWithParameters() {
+    val taco = TypeSpec.classBuilder("Taco")
```

Comment:
I believe the indentation here is incorrect, we use 2 spaces

Reply:
Fixed, thanks.